Projekty/Projekt1/Kaluska_Komorowski_Kurek/Milestone_2.ipynb

```diff
@@ -0,0 +1,734 @@
+{
```

Comment:
<p>To może być słuszne podejście, ale szkoda że nie zostało porównane z innymi. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/151/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>